README.md

```diff
@@ -6,23 +6,69 @@
 [![tests](https://github.com/sofroniewn/napari-animation/workflows/tests/badge.svg)](https://github.com/sofroniewn/napari-animation/actions)
 [![codecov](https://codecov.io/gh/sofroniewn/napari-animation/branch/master/graph/badge.svg)](https://codecov.io/gh/sofroniewn/napari-animation)
 
-A plugin to make animations with napari
+**napari-animation** is a plugin for making animations in [napari].
 
 ----------------------------------
 
 This [napari] plugin was generated with [Cookiecutter] using with [@napari]'s [cookiecutter-napari-plugin] template.
 
 It is built off of great work from @guiwitz in [naparimovie](https://github.com/guiwitz/naparimovie) which was initially submitted to napari in [PR#851](https://github.com/napari/napari/pull/780).
 
-It is currently pre-release and under active development. APIs are likely to change before it's first 0.0.1 release,
-but feedback and contributions welcome.
+----------------------------------
+## Overview
+
+**napari-animation** provides a framework for the creation of animations in napari and features:
+- an easy to use GUI for interactive creation of animations
+- Python tools for programmatic creation of animations
+
+This plugin is currently pre-release and under active development. APIs are likely to change before it's first 0.0.1 release,
+but feedback and contributions are welcome.
 
 ## Installation
 
 You can clone this repository with install locally with
 
     pip install -e .
 
+## Examples
+Examples can be found in our [examples](examples) folder. Simple examples for both interactive and headless 
+use of the plugin follow.
+
+### Interactive
+**napari-animation** can be used interactively by creating an `AnimationWidget` from a napari `Viewer` and adding it to
+the viewer as a dock widget.
+
+```python
+animation_widget = AnimationWidget(viewer)
+viewer.window.add_dock_widget(animation_widget, area='right')
+```
+
+![AnimationWidget image](resources/screenshot-animation-widget.png)
+
+### Headless
+**napari-animation** can also be run headless, allowing for reproducible, scripted creation of animations.
+
+```python
+from napari_animation import Animation
+
+animation = Animation(viewer)
+viewer.update_console({'animation': animation})
```

Comment:
```suggestion
```
We can drop that console update line. We should probably remove from examples scripts too. It's not necessary